.github/workflows/publish-snapshot.yml

```diff
@@ -22,7 +22,9 @@ jobs:
           java-version: 11.0.7
 
       - name : Publish Snapshots
-        run: ./gradlew clean build && ./gradlew publish --no-parallel --no-daemon
+        run: |
+          ./gradlew clean build --no-daemon
+          ./gradlew publish --no-parallel --no-daemon
```

Comment:
Can we count on the publish call being prevented w/o the &&?

Reply:
Yep.  The different lines are executed sequentially.  Sadly, the [documentation](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsrun) isn't very clear about this, but that's how it works.